<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1181</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pbCoke">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Coke / 1200</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbTea">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Tea / 950</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCoffee">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>150</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Coffee / 600</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbWater">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>150</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Water / 450</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb500">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb100">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb10">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>280</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb50">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>280</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber10">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>410</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber50">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>410</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber100">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>410</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber500">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>22</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbReturn">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbreSet">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>20</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
